.github/workflows/version-bump.yml

```
@@ -89,72 +80,10 @@ jobs:
       # VERSION BUMP SECTION #
       ########################
 
-      ### napi
-      - name: Bump @bitwarden/sdk-napi Version
-        if: ${{ inputs.project == 'napi' }}
-        working-directory: "crates/bitwarden-napi"
-        run: |
-          npm ci
-          npm version ${{ inputs.version_number }}
-
-      - name: Bump napi crate Version
-        if: ${{ inputs.project == 'napi' }}
-        run: cargo set-version -p bitwarden-napi ${{ inputs.version_number }}
-
       ### bitwarden
       - name: Bump bitwarden crate Version
         if: ${{ inputs.project == 'bitwarden' }}
```

I think we should also update the input and name to match:
```suggestion
      - name: Bump bitwarden-core crate Version
        if: ${{ inputs.project == 'bitwarden-core' }}
```

---

True.